tests/spark/test_mrjob_spark_harness.py

```diff
@@ -159,6 +191,14 @@ def test_sort_values_sorts_encoded_values(self):
                 dict(a=['artichoke', 'alligator', 'actuary'],
                      b=['bowling', 'balloon', 'baby']))
 
+    def test_passthru_args(self):
+        input_bytes = b'\n'.join([
+            b'to be or',
+            b'not to be',
+            b'that is the question'])
 
+        job = self._harness_job(
+            MRSortAndGroupReversedText, input_bytes=input_bytes,
```

Comment:
Confused, I thought this would run MRPassThruArgs, which seems to be unused.

Reply:
oops, I forget to change it back while testing some other stuff.  updated!